### PR TITLE
ActiveMQ (STOMP) support

### LINKED ISSUE #9: Support for ActiveMQ
As requested on /r/devops

### PR DESCRIPTION
**Pull request by blinktag**
Resolves #9

### CODE REVIEW

File: backends/activemq/activemq.go [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	log     *logrus.Entry
+}
+
+// NewClient returns a configured instance of stomp.Conn
+func NewClient(opts *cli.Options) (*stomp.Conn, error) {
+	o := func(*stomp.Conn) error {
+		return nil
+	}
+
+	conn, err := stomp.Dial("tcp", "localhost:61613", o)
```
Comment (dselans) [inline backends/activemq/activemq.go:25]: This should be dialing the opts field.

File: cli/activemq.go [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	addSharedActiveMqFlags(rc, opts)
+
+	wc := writeCmd.Command("activemq", "ActiveMQ STOMP")
+	addSharedActiveMqFlags(wc, opts)
+}
+
+func addSharedActiveMqFlags(cmd *kingpin.CmdClause, opts *Options) {
+	cmd.Flag("address", "Destination host address").Default("tcp://localhost:61613").StringVar(&opts.ActiveMq.Address)
+	cmd.Flag("topic", "Topic to read message(s) from").StringVar(&opts.ActiveMq.Topic)
+	cmd.Flag("queue", "Topic to read message(s) from").StringVar(&opts.ActiveMq.Queue)
```
Comment (dselans) [inline cli/activemq.go:29]:
```suggestion
	cmd.Flag("queue", "Queue to read message(s) from").StringVar(&opts.ActiveMq.Queue)
```

**Review by dselans — CHANGES_REQUESTED** [review]
Two minor fixes. Looking great!

File: backends/activemq/activemq.go (hunk 1 above)
Comment (blinktag) [reply backends/activemq/activemq.go]: Fixed

**Review by dselans — APPROVED** [review]

**Pull request merged**